bylaws/003-membership.md

```diff
@@ -98,9 +98,13 @@ will be assumed.
 
 If, in the judgement of PHP-FIG, a Voting Representative is acting
 inappropriately and to the detriment of PHP-FIG's ability to meet its
-objectives, a vote may be taken to request a replacement Voting Representative in
-accordance with the [Voting Protocol bylaw][voting] or to expel the Member
-Project where replacing a Voting Representative is not possible.
+objectives, a vote may be taken to request a replacement Voting Representative,
+or to expel the Member Project where replacing a Voting Representative is not
```

Comment:
Ok! I’ll have a go when I’m done with my talk! Wish me luck :) 

-- 
Phil

On May 28, 2016 at 4:00:57 PM, Michael C. (notifications@github.com) wrote:

In bylaws/003-membership.md:

> @@ -98,9 +98,13 @@ will be assumed.
>   
>  If, in the judgement of PHP-FIG, a Voting Representative is acting
>  inappropriately and to the detriment of PHP-FIG's ability to meet its
> -objectives, a vote may be taken to request a replacement Voting Representative in
> -accordance with the [Voting Protocol bylaw][voting] or to expel the Member
> -Project where replacing a Voting Representative is not possible.
> +objectives, a vote may be taken to request a replacement Voting Representative,
> +or to expel the Member Project where replacing a Voting Representative is not
That doesn't read particularly well I feel.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.